Sources/RInAppMessaging/Views/FullView.swift

```diff
@@ -101,6 +101,13 @@ internal class FullView: UIView, FullViewType, RichContentBrowsable {
         super.layoutSubviews()
 
         updateUIConstants()
+        
+        // Clickable Image Feature only for RMC
+        if RInAppMessaging.isRMCEnvironment {
+            imageView.isUserInteractionEnabled = true
```

Comment:
Where are making isUserInteractionEnabled = false

Reply:
By default its false. Its only enabled for RMC-IAM. Rest of the validation is in didClickCampaignImage()